setup.py

```diff
@@ -73,7 +73,7 @@ def get_version(package):
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['PyJWT', 'requests', 'pubnub<4.0.0', 'pyotp'],
+    install_requires=['PyJWT>=1.5.0', 'requests>=2.19.1', 'pyotp>=2.2.5', 'pyOpenSSL==18.0.0', 'Twisted>=18.7.0', 'service-identity==17.0.0', 'google-auth-oauthlib>=0.2.0'],
```

Comment:
what do we need `google-auth-oauthlib` and `service-identity` for?

Reply:
pyOpenSSL, service-identity, google-auth-oauthlib are dependencies for twisted, otherwise, we will see some ugly SSL certificate warnings when sending https request.